build/build.proj

```diff
@@ -0,0 +1,100 @@
+<Project DefaultTargets="PrepareMachine;RestoreToolset;CallToolset;GenerateDependentAssemblyVersionsFile;UploadCodeCoverage">
```

Comment:
I take it these targets will be run in the given order?

Is there an advantage to this over having a `Build` target that depends on these five? Like so:
``` XML
<Project DefaultTargets="Build">
...
  <Target Name="Build"
          DependsOn="PrepareMachine;RestoreToolset;CallToolset;GenerateDependentAssemblyVersionsFile;UploadCodeCoverage">
  ...
  </Target>
</Project>
```

Reply:
I used to do what you are suggesting, but realized that DefaultTargets can be multiple and felt it was a little simpiler.